cmd/util.go

```diff
@@ -45,23 +44,23 @@ var (
 	sanitizeID = strings.NewReplacer("_", "-")
 )
 
-type exportConfig struct {
-	euiForID    bool
-	devIDPrefix string
+type ExportConfig struct {
```

Comment:
I think this can stay unexposed if we parse the parameters in the individual subcommands. You can even consider moving this config + parsing to an individual package that others may import without creating cycles or headaches.

Reply:
I moved the export part to `pkg/export`